lib/views/oauth_page.dart

```diff
@@ -141,7 +142,7 @@ class OAuthPageState extends State<OAuthPage> {
       return await authenticate(context);
     }
 
-    return oauthModel.accessToken != null;
+    return false;
```

Comment:
Are you sure of this one ?

Reply:
@taorepoara 
Well because the access token can be expired or invalid, we cannot rely on it to authenticate the user. 

Either we force the user to authenticate if the LocalStorage does not contain a Token, which is what I have done here
OR
We make sure that the accessToken from the oauthModel is valid before returning `oauthModel.accessToken != null`

Reply:
@taorepoara  But I'm thinking that we can also just let the guard return true in this case and let the server return an error `not authenticated` which will redirect us to the oauth page.

Reply:
What do you think ?

Reply:
This change will probably call too much our API. We should be careful with this...

Can you make some tests ?